libs/datasets/sources/nytimes_dataset.py

```diff
@@ -1,18 +1,18 @@
-from typing import List
-import logging
-import numpy
 import pandas as pd
 from libs.datasets.timeseries import TimeseriesDataset
 from libs.datasets import data_source
 from libs.datasets import dataset_utils
 from libs.datasets.common_fields import CommonIndexFields
-from libs.datasets.common_fields import CommonFields
```

Comment:
Ah, this is a bit of cleanup for code I wrote, but could you replace `TimeseriesDataset.Fields` with `CommonFields` in the `COMMON_FIELD_MAP` below? 

Just would like to keep that consistent if possible!  It is true that the `TimeseriesDataset.Fields` inherits common fields I believe, but just using `CommonFields` makes it a bit more explicity.

Reply:
sure, will do!